### PR TITLE
DISTX-546 Enable load-balancer for 7.2.9 and 7.2.10 by default

### PR DESCRIPTION
**Pull request by cegganesh84**
1. 7.2.8 is full tested in dev with both public and private subnets individually.
2. Promoting this change for all the future releases.
3. The template is still in preview state.

### CODE REVIEW

File: core/src/main/resources/defaults/clustertemplates/7.2.9/aws/dataengineering-ha.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "externalDatabase": {
       "availabilityType": "HA"
     },
+    "enableLoadBalancer": true,
```
Comment (sidseth) [inline core/src/main/resources/defaults/clustertemplates/7.2.9/aws/dataengineering-ha.json:14]: Along with this, does the manager need to be changed to type CORE, and the master to GATEWAY. That is at least what 7.2.8 does. Unless that change is expected in the change @smaniraju is currently making.

Comment (cegganesh84) [reply core/src/main/resources/defaults/clustertemplates/7.2.9/aws/dataengineering-ha.json:14]: Yeah, I am relying on @smaniraju's change.

Comment (cegganesh84) [reply core/src/main/resources/defaults/clustertemplates/7.2.9/aws/dataengineering-ha.json:14]: Yes, particularly I did not want to introduce merge conflicts for @smaniraju

File: core/src/main/resources/defaults/clustertemplates/7.2.10/aws/dataengineering-ha.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
     "externalDatabase": {
       "availabilityType": "HA"
     },
+    "enableLoadBalancer": true,
```
Comment (doktoric) [inline core/src/main/resources/defaults/clustertemplates/7.2.10/aws/dataengineering-ha.json:14]: I thin kthis feature is under entitlement

Comment (cegganesh84) [reply core/src/main/resources/defaults/clustertemplates/7.2.10/aws/dataengineering-ha.json:14]: No Richard. This is like DL HA LB. No entitlement. Unline DL HA, DE HA template is still in "PREVIEW" state.

**Review by doktoric — APPROVED** [review]

**Pull request merged**